modules/apps/packet-forward-middleware/ibc_middleware_test.go

```diff
@@ -0,0 +1,254 @@
+package packetforward_test
+
+import (
+	"encoding/json"
+	"log"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	packetforward "github.com/cosmos/ibc-go/v10/modules/apps/packet-forward-middleware"
+	packetforwardkeeper "github.com/cosmos/ibc-go/v10/modules/apps/packet-forward-middleware/keeper"
+	packetforwardtypes "github.com/cosmos/ibc-go/v10/modules/apps/packet-forward-middleware/types"
+	transfertypes "github.com/cosmos/ibc-go/v10/modules/apps/transfer/types"
+	channeltypes "github.com/cosmos/ibc-go/v10/modules/core/04-channel/types"
+	ibctesting "github.com/cosmos/ibc-go/v10/testing"
+)
+
+// CallbacksTestSuite defines the needed instances and methods to test callbacks
```

Comment:
```suggestion
```

For one the name was wrong, but the comment didn't really add any value to my eye, so might as well just drop it